ablySpec/RealtimeClientChannel.swift

```diff
@@ -950,7 +950,7 @@ class RealtimeClientChannel: QuickSpec {
                             expect(messageSent.messages![0].clientId).to(beNil())
 
                             let messageReceived = transport.protocolMessagesReceived.filter({ $0.action == .Message })[0]
-                            expect(messageReceived.messages![0].clientId).to(beNil())
+                            expect(messageReceived.messages![0].clientId).to(equal(options.clientId))
```

Comment:
I don't agree with this change. It should be nil...

[RTL6g1b](http://docs.ably.io/client-lib-development-guide/features/#RTL6g1a)
> the clientId value is null for the Message when received